otx/algorithms/classification/adapters/mmcls/models/heads/contrastive_head.py

```diff
@@ -0,0 +1,54 @@
+# pylint: disable=missing-module-docstring, unused-argument
```

Comment:
Adding short description would be more propriate than disable pylint

Reply:
Done.